app/plugins/ee-openid-connect.md

```diff
@@ -0,0 +1,130 @@
+---
+id: page-plugin
+title: Plugins - OpenID Connect
+header_title: OpenID Connect
+header_icon: /assets/images/icons/plugins/ee-openid-connect.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Requesting Access
+---
+
+OpenID Connect ([1.0][connect]) plugin allows the integration with a 3rd party
+identity provider (IdP) or [Kong OAuth 2.0 Plugin][oauth2plugin] in a standardized way.
+This plugin can be used to implement Kong as a (proxying) [OAuth 2.0][oauth2] resource
+server (RS) and / or as a OpenID Connect relying party (RP) between the client
+and the upstream service.
+
+The plugin supports several types of credentials. Currently it can be used with
+the following types of credentials:
+
+- Signed [JWT][jwt] access tokens ([JWS][jws]) with the standardised signing algorithms ([JWA][jwa])
```

Comment:
We tend to follow US English in our documentation, and as such we should spell this "standardized"